runtime/LibcWrappers.cpp

```diff
@@ -246,6 +267,32 @@ size_t SYM(fread)(void *ptr, size_t size, size_t nmemb, FILE *stream) {
   return result;
 }
 
+size_t SYM(fread_unlocked)(void *ptr, size_t size, size_t nmemb, FILE *stream)
+{
+    tryAlternative(ptr, _sym_get_parameter_expression(0), SYM(fread_unlocked));
+    tryAlternative(size, _sym_get_parameter_expression(1), SYM(fread_unlocked));
+    tryAlternative(nmemb, _sym_get_parameter_expression(2), SYM(fread_unlocked));
+
+    auto result = fread(ptr, size, nmemb, stream);
```

Comment:
I wonder if aliasing to fread (the thread safe function) would prevent detecting some synchronisation bugs? Not really a use case for SymCC now, but it could be. 
Why not calling fread_unlocked ?

Reply:
Revised